resources/views/checkout/information.antlers.html

```diff
@@ -32,10 +32,10 @@ <h2 class="text-gray-900 font-bold text-2xl mb-4">
 
                         <div class="grid md:grid-cols-12 gap-6 mb-12">
                             {{ partial:components/forms/input_field width="50" handle="shipping_name" display="Shipping Name" required="true" }}
-                            {{ partial:components/forms/input_field width="50" handle="shipping_address_line1" display="Address Line 1" required="true" }}
+                            {{ partial:components/forms/input_field width="50" handle="shipping_address" display="Address Line 1" required="true" }}
```

Comment:
In fact.. this one should probably be changed on the order blueprint side instead.